explorer/client/src/components/ownedobjects/OwnedObjects.tsx

```diff
@@ -0,0 +1,248 @@
+    } else {
+        return <div />;
+    }
+}
+
+function OwnedObjectView({ results }: { results: resultType }) {
+    const handlePreviewClick = useCallback(
+        (id: string, navigate: Function) => (e: React.MouseEvent) =>
+            navigateWithUnknown(id, navigate),
+        []
+    );
+    const navigate = useNavigate();
+    return (
+        <div id="ownedObjects" className={styles.ownedobjects}>
+            {results.map((entryObj, index1) => (
+                <div
+                    className={styles.objectbox}
+                    key={`object-${index1}`}
+                    onClick={handlePreviewClick(entryObj.id, navigate)}
+                >
+                    {entryObj.display !== undefined ? (
+                        <div className={styles.previewimage}>
+                            <DisplayBox
+                                display={entryObj.display}
+                                tag="imageURL"
+                            />
+                        </div>
+                    ) : (
+                        <div className={styles.previewimage} />
+                    )}
+                    {Object.entries(entryObj).map(([key, value], index2) => (
+                        <div key={`object-${index1}-${index2}`}>
+                            {(() => {
+                                switch (key) {
+                                    case 'display':
+                                        break;
+                                    case 'Type':
+                                        return (
+                                            <div>
+                                                <span>{key}</span>
+                                                <span>
+                                                    {typeof value === 'string'
+                                                        ? trimStdLibPrefix(
+                                                              value
+                                                          )
+                                                        : ''}
+                                                </span>
+                                            </div>
+                                        );
+                                    default:
+                                        return (
+                                            <div>
+                                                <span>{key}</span>
+                                                <span>{value}</span>
+                                            </div>
+                                        );
+                                }
+                            })()}
+                        </div>
+                    ))}
+                </div>
+            ))}
+        </div>
+    );
+}
+
+function OwnedObject({ objects }: { objects: string[] }) {
+    const [pageIndex, setPageIndex] = useState(0);
+
+    const ITEMS_PER_PAGE = 4;
```

Comment:
4 items per page is pretty sparse, it's harder to get a good overview of an address with such small pages. Can we make this 8, 10, even 12?

Reply:
The number of results per page have been increased to 12. This required updating the mock JSON dataset and the tests.